lib/items.ts

```diff
@@ -51,7 +54,7 @@ export function checkCorrect(
     return i.id === item.id;
   });
 
-  if (index !== correctIndex) {
+  if (index !== correctIndex && item.year !== [...played, item][correctIndex].year) {
```

Comment:
It's best to keep PRs to changing one thing at a time. I would remove this and the discussion can be kept on https://github.com/tom-james-watson/wikitrivia/pull/74

Reply:
Of course, that makes sense. I thought this would make it easier to merge, but I can see how it makes it more confusing.  This has been reverted.